coralproject/talk
# [CORL-2682] Make download comments title a heading

## What does this PR do?
This PR makes the title for "Download My Comment History" a proper `<h1>` element, which is better for accessibility.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Enable comment history downloads
2. In the stream, navigate to My Profile > My Comments
3. Observe that "Download My Comment History" is an h1 rather than a div
 
## How do we deploy this PR?
No special considerations should be nessecary.
